features/gateway_injection.md

```diff
@@ -0,0 +1,146 @@
+# Feature: Gateway Injection
+
+**Feature name:** Gateway Injection
+
+**Primary lead(s):** @howardjohn
+
+**Short description:** Injecting gateway deployments to enabled minimal configuration and simplified operations.
+
+---
+
+## Experimental
+
+### Requirements:
+
+- [ ] [User stories](insert_your_link_here) reviewed in a work group meeting.
+
+- [x] [RFC Authored] - [Injector V2](https://docs.google.com/document/d/1Rmp9B3DDypgMCau-YAqidx_r3qvKLZj6jUX-t22zj84/).
+
+- [x] [Documentation](https://preliminary.istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental) and [here](https://preliminary.istio.io/latest/docs/setup/additional-setup/external-controlplane/) for enabling and using the feature.
+
+- [ ] [Feedback plan](insert_your_link_here). ??
```

Comment:
This one is really only useful for Experimental. Designed so that when we create experimental features, we have an idea of how we will tell if this feature is useful for users and if anything needs to change before it goes alpha.